modules/tinymce/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "rootDir": "src",
     "baseUrl": ".",
     "tsBuildInfoFile": "lib/tsconfig.tsbuildinfo",
+    "allowSyntheticDefaultImports": true,
```

Comment:
I needed this due to how the dompurify types had been written — let me know if this is going to cause issues, because there are a lot of build tools we use that this might interfere with.